CRM/Report/Form/Contact/Detail.php

```diff
@@ -898,25 +897,21 @@ public function alterDisplay(&$rows) {
   /**
    * @param array $componentRows
    */
-  public function alterComponentDisplay(&$componentRows) {
-    // custom code to alter rows
-    $activityTypes = CRM_Core_PseudoConstant::activityType(TRUE, TRUE, FALSE, 'label', TRUE);
-    $activityStatus = CRM_Core_PseudoConstant::activityStatus();
-
+  public function alterComponentDisplay(array &$componentRows): void {
     $entryFound = FALSE;
     foreach ($componentRows as $contactID => $components) {
       foreach ($components as $component => $rows) {
         foreach ($rows as $rowNum => $row) {
           // handle contribution
-          if ($component == 'contribution_civireport') {
+          if ($component === 'contribution_civireport') {
             $val = $row['civicrm_contribution_financial_type_id'] ?? NULL;
             if ($val) {
-              $componentRows[$contactID][$component][$rowNum]['civicrm_contribution_financial_type_id'] = CRM_Contribute_PseudoConstant::financialType($val, FALSE);
+              $componentRows[$contactID][$component][$rowNum]['civicrm_contribution_financial_type_id'] = CRM_Core_PseudoConstant::getLabel('CRM_Contribute_BAO_Contribution', 'contribution_status_id', $val);
```

Comment:
@eileenmcnaughton This looks wrong to me

Reply:
the word status perchance?